### PR TITLE
Transaction endpoint

### PR DESCRIPTION
**Pull request by j-kon**
**What does this PR do?**

A summary of what this PR is about

**List of task completed**

-   [x] Specify what tasks are completed in this PR
-   [ ] Specify what's left to be done

**How can this be tested?**

- Highlight how this PR can be tested

**Additional info**

Add more info if any. eg fixes #.

### CODE REVIEW

File: Frontend/lib/data/api/api_handlers/cosign_api.dart [hunk 1]
```diff
@@ -21,7 +21,7 @@ class CoSignApi {
           queryParameters: queryParameters,
           options: token == null
               ? null
-              : Options(headers: {'Authorization': 'Bearer $token'}));
+              : Options(headers: {'Authorization': 'Token $token'}));
 
       return ApiUtils.toApiResponse(response);
     } on DioError catch (e) {
```
Comment (Muneerah1612) [inline Frontend/lib/data/api/api_handlers/cosign_api.dart:27]: Just curious, what is the meaning of DioError?

File: Frontend/lib/data/models/models.dart [hunk 2]
```diff
@@ -1,9 +1,12 @@
 export 'address_info.dart';
 export 'address_list.dart';
+export 'broadcast_transaction.dart';
+export 'create_transaction.dart';
 export 'generate_address.dart';
 export 'get_all_transaction.dart';
 export 'get_new_address.dart';
 export 'total_utxo.dart';
+export 'total_utxo.dart';
```
Comment (Muneerah1612) [inline Frontend/lib/data/models/models.dart:9]: Why did you export total_utxo again, you already exported it

Comment (Muneerah1612) [inline Frontend/lib/data/models/models.dart:9]: Why did you export total_utxo again, you already exported it

File: Frontend/lib/screens/address/address_list.dart [hunk 3]
```diff
@@ -9,8 +12,23 @@ class AddressList extends StatefulWidget {
 }
 
 class _AddressListState extends State<AddressList> {
+  final titles = ["List 1", "List 2", "List 3"];
+  final subtitles = [
```
Comment (Muneerah1612) [inline Frontend/lib/screens/address/address_list.dart:16]: what does this do?

File: Frontend/lib/utils/constant.dart [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import 'package:form_field_validator/form_field_validator.dart';
 
-const baseUrl = 'http://localhost:8000/api/v1';
+const baseUrl = 'https://co-sign.herokuapp.com/api/v1';
```
Comment (Muneerah1612) [inline Frontend/lib/utils/constant.dart:3]: I  thought we agreed you will use docker,anyway just to be sure,are you using the heroku version or the docker version?

**Review by Muneerah1612 — COMMENTED** [review]
Looks good to me

File: Frontend/lib/data/models/models.dart (hunk 2 above)
Comment (j-kon) [reply Frontend/lib/data/models/models.dart:9]: That must be a typo, I will make the correction.

Comment (j-kon) [reply Frontend/lib/data/models/models.dart:9]: That must be a typo, I will make the correction.

File: Frontend/lib/utils/constant.dart (hunk 4 above)
Comment (j-kon) [reply Frontend/lib/utils/constant.dart:3]: It has been removed from the other branch.

File: Frontend/lib/screens/address/address_list.dart (hunk 3 above)
Comment (j-kon) [reply Frontend/lib/screens/address/address_list.dart:16]: I will remove this, I was trying to hardcode for listing.

File: Frontend/lib/data/api/api_handlers/cosign_api.dart (hunk 1 above)
Comment (j-kon) [reply Frontend/lib/data/api/api_handlers/cosign_api.dart:27]: Dio is the name of the package and a powerful Http client for Dart.

File: Frontend/lib/utils/constant.dart (hunk 4 above)
Comment (Muneerah1612) [reply Frontend/lib/utils/constant.dart:3]: which branch?

File: Frontend/lib/screens/address/address_list.dart (hunk 3 above)
Comment (Muneerah1612) [reply Frontend/lib/screens/address/address_list.dart:16]: oh okay

File: Frontend/lib/utils/constant.dart (hunk 4 above)
Comment (j-kon) [reply Frontend/lib/utils/constant.dart:3]: address endpoint.

Comment (Muneerah1612) [reply Frontend/lib/utils/constant.dart:3]: Okay, make sure you are testing with the Heroku version also.

**Review by Muneerah1612 — APPROVED** [review]

**Pull request merged**